### PR TITLE
Some UI updates

### PR DESCRIPTION
**Pull request by mattish91**
Im sure some of these updates are welcome to the comunity and since i had some sparetime and have been using Ampache ALOT durring my development time in PHP, HTML, CSS, XML, LUA and also some JS and Java, i decided that maybe, just maybe i could change some elements of the UI to make it look a little bit cleaner, so i did, and here it is :D

Working flawlessly in version 3.9.1-dev branch and also works well with 3.9.0 stable branch. 

The default thread about it in google groups: https://groups.google.com/forum/#!topic/ampache/NVH84Xm8SrQ

Im also going to release these files for current 3.9.0 users to upgrade their own files in case they want to change the element, it seriously changes more of the feel to the player than anything else.

### CODE REVIEW

**Comment by lachlan-00** [comment]
Hey that's awesome. I have been debating about what to do with the player a lot and this looks really solid.

Hold off on releasing files as I'm in the middle of rebasing the themes in core.
What I'm going to do is alter this to pull into core first and I'll be working on better theming docs before it goes into develop.

Thank you! This is really cool

**Comment by lachlan-00** [comment]
One thing to add. In the new branch I've added $_SESSION['mobile'] to the code so you are able to show/hide things in the interface based on that as well

**Comment by mattish91** [comment]
Alright! i pull the dl links from the group, is there anything i might be able to help with?

**Comment by mattish91** [comment]
Is this in develop branch? Ill have a look at it right away.

**Comment by kuzi-moto** [comment]
Core should be the newest branch. But it will be merged into develop pretty soon.

**Comment by mattish91** [comment]
Oh xD i basically just copied develop branch straight off xD ill have a look into core and see what i can do there then ^^

**Comment by lachlan-00** [comment]
Yeah @mattish91 core is where I've been working. The themes/interface are finished from what I wanted to achieve so if you do a pr on that branch I can merge it there instead.

I'll do a write up about themes in the wiki and tag you when I finish.

Basically the structure in themes should be
* theme default CSS is structure/size/etc
* light/dark/accent CSS is colours/visual effects/non-structural
* Icons/images are done from the images folder instead of CSS. (Unless they are colour specific)

**Comment by mattish91** [comment]
as of what i can see, there is basically no major changes to any of these files. Im comparing them in NPP with only a small set of changes to edit. Not too sure how drastically it will be to edit in tho.

**Comment by mattish91** [comment]
Yea, ill do it, I will try and merge in the updated UI into the core branch, but it will probably take me a while. Im also wondering if it would be all right to actually remove those parts where i simply comented out in my edit and maybe document that a bit as of why i removed stuff. or do you need to keep all features in current nav as is? (Search and header specific) since most of those stuff are located in advanced search already and i have never used some of those features. 

However, more improvements could be made here, a dropdown from username to access admin, and user related stuff instead of top right bar. this would also make more sense on a mobile platform where dropdowns are more common and related. 

In core branch i can't see any mobile related UI as of yet, was that to be expected? xD

Great job so far on everything tho! 

Funny how i some how happened to get 3 installs of ampache on my dev server today xD
-stable
-develop
-core

**Comment by lachlan-00** [comment]
Only a few mobile related changes which are in code mostly instead of CSS.
* Moving the logo to the bottom on login/reset password
* Hide search button in top bar
* Hide action buttons on webplayer

I've also added viewports to the pages to allow better view on mobile.

I haven't tried to change a lot visually just cleaning and arrangements to make it easier for others.

The current core reborn is going to end up in all branches soon so now is the time to make changes.

(The one thing I hate is the share page. I don't like it on web/mobile.)

File: templates/header.inc.php [hunk 1]
```diff
@@ -356,17 +357,18 @@ function libitem_action(item, action)
...
                   <a href="<?php echo $web_path; ?>/index.php">
-                    <img src="<?php echo UI::get_logo_url(); ?>" title="<?php echo AmpConfig::get('site_title'); ?>" alt="<?php echo AmpConfig::get('site_title'); ?>" />
+                    <img style="width:48px; height:48px;" src="<?php echo UI::get_logo_url(); ?>" title="<?php echo AmpConfig::get('site_title'); ?>" alt="<?php echo AmpConfig::get('site_title'); ?>" />
                   </a>
                 </h1>
-                <div id="headerbox">
-                    <?php
-                        UI::show_box_top('', 'box box_headerbox');
+                    <?php 
+                        #UI::show_box_top('', 'box box_headerbox');
```
Comment (lachlan-00) [inline templates/header.inc.php]: Comment with // instead of # in the php

File: templates/show_song_row.inc.php [hunk 2]
```diff
@@ -53,7 +53,7 @@
         <a id="<?php echo 'add_playlist_' . $libitem->id ?>" onclick="showPlaylistDialog(event, 'song', '<?php echo $libitem->id ?>')">
             <?php echo UI::get_icon('playlist_add', T_('Add to existing playlist')); ?>
         </a>
-    <?php
+    <?php
```
Comment (lachlan-00) [inline templates/show_song_row.inc.php]: in the code dir run codestyle fixes over the changes
```
./php-cs-fixer -v fix
```

**Comment by lachlan-00** [comment]
Fyi. core is merged into develop now! :)

**Comment by mattish91** [comment]
Great, ill start working on it later tonight!

**Comment by mattish91** [comment]
I got about an hour over right now, so ill start to merge my changes in to the new develop branch locally first. just to iron out any bugs and stuff like that. ill probably push some changes later today/tomorrow

**Comment by mattish91** [comment]
Hey! did you merge it already or is my cache spooking me again? :O

**Comment by mattish91** [comment]
i just made a new pull, think this didn't actually update to your latest merge. way to many changes got pulled in to this.

**Comment by lachlan-00** [comment]
Let me look at it in git desktop and see what needs to be merged.

**Comment by mattish91** [comment]
Next update: mobile UI in the works! all CSS based as well! 

Have a look at it: https://www.mspot.nu/files/public/488317

**Comment by lachlan-00** [comment]
Lot's of conflicts from missing the last core update into develop. nothing big it's just conflicts from the same content mostly.

I think the easiest way to fix this would be if you used github desktop to merge the current develop branch or restart on a different branch. and paste over the top.

**Comment by lachlan-00** [comment]
Ack! still conflicted.

@mattish91 i'm sorry for pasting over your work but i'm just trying to bring it up to current develop.

My only two concerns about the theme so far are relying on external css for anything and if we make this theme a style on reborn it should hide elements in the css instead of in the template code.

The reason i'd hide it is so we can allow people to keep the old reborn theme style as well.

**Comment by mattish91** [comment]
No worries, the only reason i implemented that fontawesome css were only if i were to replace the player buttons which i never came to do, it's safe to remove that as of now. The thing with the buttons is that they don't scale well, that's the only reason i wanted to change it. So that code is 100% safe to remove from header.inc.php if you want.

**Comment by mattish91** [comment]
@lachlan-00 this were more of a concept in the beginning of the development and not so much as full ready to use. If you are willing to improve on it, feel free to do so. Im not going to haunt you for that ^^ i have been using Ampache for almost 5 years now and just wanted to give something back to the community.

**Comment by lachlan-00** [comment]
The buttons are very much holdovers from the early 00's (well before SVGs!) What i do like about the icons is that they are easily replaceable as far as a theme goes. the only issue is that when they are part of a theme they are part of each scheme.

What I might do is make a branch for this as an updated theme choice. (dark|light|responsive) That way anyone can play with it and it should be able to merge right into any branch from there.

Oh i'm also calling your theme 'responsive' ;)

**Comment by mattish91** [comment]
Since most of this seams to be scrap in the new version, im simply going to start it over someday soon, i also noticed theme support is a thing that seams to be working now in current develop, so im just going to take some advantage from that to not break the current working reborn theme at all.

**Comment by mattish91** [comment]
also noticed that cache is a pain in the ass when i have to reload every css file by hand -.- figuring i should just temporarily add a no-cache in the header of my current install xD

**Comment by mattish91** [comment]
chromecast support should be something to look into by the way (web version for desktop, only in chrome), seams fairly easy from what i have read. https://developers.google.com/cast/docs/developers

**Comment by lachlan-00** [comment]
lets move this into ampache/pull/2038

I'm excited to see what you can do with a bit of freedom and svg support!

**Pull request closed without merging**